book/src/integrations/radius.md

```diff
@@ -200,14 +144,17 @@ the problem. To increase the logging level you can re-run your environment with
 ```bash
 docker rm radiusd
 docker run --name radiusd \
-    -e DEBUG=True \
+    --rm -e DEBUG=True \
     -p 1812:1812 \
-    -p 1812:1812/udp
+    -p 1812:1812/udp \
     --interactive --tty \
-    --volume /tmp/kanidm:/etc/raddb/certs \
+    --mount "type=bind,src=$(pwd)/examples/radius.toml,target=/data/kanidm" \
+    --mount "type=bind,src=/tmp/kanidm,target=/data" \
```

Comment:
`--mount "type=bind,..."` is more explicit but any other reason why we aren't recommending `-v "$(pwd)/examples/radius.toml:/data/kanidm"` especially given how much cleaner it is?

Reply:
'cause I hate the non-explicit binds that are clear exactly what's going on 😄

Reply:
did you mean to write "aren't" rather than "are"? Sounds like an endorsement for `-v` :P